client/gui/CGuiHandler.cpp

```diff
@@ -63,6 +64,8 @@ void CGuiHandler::init()
 	shortcutsHandlerInstance = std::make_unique<ShortcutHandler>();
 	inputHandlerInstance = std::make_unique<InputHandler>(); // Must be after windowHandlerInstance and shortcutsHandlerInstance
 	framerateManagerInstance = std::make_unique<FramerateManager>(settings["video"]["targetfps"].Integer());
+
+	AssetGenerator::generate();
```

Comment:
Should be done on first access to corresponding image, to avoid increasing startup time

Reply:
Yes. But not sure, how to improve this without changes in the resource system.

At the moment for the two resources it's not a big problem as resource generation is quite fast.

Reply:
On my laptop-cpu inside of a VM it needs 10ms currently.

Reply:
1 ms here, 1 ms there...
And on low-end phone this would be not 10 ms but 50 ms or even more.

Reply:
now they will only generated when needed...